deposit_2.py

```diff
@@ -0,0 +1,67 @@
+"""Создайте класс вклад. Который содержит необходимые
+ поля и методы, например сумма вклада и его срок.
+ Пользователь делает вклад в размере N рублей сроком
+ на R лет под 10% годовых (вклад с возможностью ежемесячной
+  капитализации - это означает, что проценты прибавляются
+   к сумме вклада ежемесячно). Написать класс Bank,
+   метод deposit принимает аргументы N и R, и
+   возвращает сумму, которая будет на счету пользователя."""
+
+
+class Deposit:
+    """
+    Represents a deposit.
+    """
+
+    def calculate_final_money(self, money: float, duration: int,
+                              annual_rate: float = 0.25) -> float:
+        """
+        Calculates the total deposit amount taking
+        into account the monthly capitalization of interest.
+        """
+        m = money
+        n = 12
+        y = duration
+        r = annual_rate
+        a = m * (1 + r / n) ** (n * y)
+        return a
```

Comment:
You do not need to create so many variables:
```suggestion
return money * (1 + annual_rate / n) ** (n * duration)
```

Reply:
fixed, ty